legacy/pom.xml

```diff
@@ -70,14 +70,26 @@
     <dependency>
       <groupId>com.amazonaws</groupId>
       <artifactId>aws-java-sdk</artifactId>
-      <version>1.0.008</version>
+      <version>${dependency.aws-java-sdk.revision}</version>
       <exclusions>
         <exclusion>
           <artifactId>jackson-core-asl</artifactId>
           <groupId>org.codehaus.jackson</groupId>
         </exclusion>
       </exclusions>
     </dependency>
+    <dependencies>
```

Comment:
Oops. You are adding an opening <dependencies> tag in the middle of the dependencies here.